src/qt/intro.cpp

```diff
@@ -361,6 +381,11 @@ void Intro::UpdatePruneLabels(bool prune_checked)
         storageRequiresMsg = tr("Approximately %1 GB of data will be stored in this directory.");
     }
     ui->lblExplanation3->setVisible(prune_checked);
+    ui->pruneGB->setEnabled(prune_checked);
+    static constexpr uint64_t nPowTargetSpacing = 10 * 60;  // from chainparams, which we don't have at this stage
+    static constexpr uint32_t expected_block_data_size = 2250000;  // includes undo data
+    static const uint64_t expected_backup_days = m_prune_target_gb * 1e9 / (uint64_t(expected_block_data_size) * 86400 / nPowTargetSpacing);
```

Comment:
setting `expected_backup_days` as a `static const` means that the number of days the prune target is good for will never update as the prune target is changed.
```suggestion
    uint64_t expected_backup_days = m_prune_target_gb * 1e9 / (uint64_t(expected_block_data_size) * 86400 / nPowTargetSpacing);
```

Reply:
Fixed (const can remain)